### ISSUE TITLE
WCAG Include

### ISSUE THREAD

**Issue by roelvangils** [ThePacielloGroup/infusion#11]
That WCAG thing is really cool! I did notice that `{{% wcag include="1.3.1, 4.1.2" descriptions="false" %}}`still renders the descriptions though. 

I think these should have the same effect:

```
{{% wcag include="1.3.1, 4.1.2" descriptions="false" %}}
{{% wcag include="1.3.1, 4.1.2" %}}
```

**Comment by Heydon** [comment]
@roelvangils Yeah, at the moment the descriptions are placed in there if the attribute is included at all, with any value. Will look into catching the "false" keyword to.

**Comment by Heydon** [comment]
@roelvangils Okay, false values are now honored, thanks to a3cfd63.

Thanks for the suggestion and I hope you are enjoying using Infusion :-)

**Comment by roelvangils** [comment]
Yes, I'm enjoying it!

I'm using it now to build a little best practices guide for a client (a bank). Initially, they wanted me to use (their) Confluence for this, but that doesn't feel right.

**Comment by Heydon** [comment]
@roelvangils Oh, that's great to hear.
